compiler/luci/import/src/CircleImportMetadata.cpp

```diff
@@ -21,8 +21,10 @@
 namespace
 {
 
-uint32_t read_u32(const std::vector<uint8_t> &buffer, uint32_t idx)
+template <typename VectorType> uint32_t read_u32(const VectorType &buffer, uint32_t idx)
```

Comment:
`typename VectorType` : `VectorType` makes confusion reading the code. I was searching `VectorType` at first sight.
 It would be better to use `VECTORTYPE`

Reply:
This seems one of the changes that don't fit in with `Use direct access API` or is it?

Reply:
It's necessary to change  this function to work with `VectorWrapper` instead `std::vector`. I see 3 ways to do that:

1. Just change type of argument (`std::vector<uint8>` -> `VectorWrapper<uin8_t>`). 
2. Add copy of this function with `VectorWrapper` as argument. There is a lot of new lines will be added =(
3. Make this function as template.

I chose №3, but I think we could take approach №1 too, if this option looks better for rerview.

Reply:
OK thanks for the explanation. So to use `VectorWrapper` which I understand as good helper, this change is also a good helper. I think we can go as is like the draft code here.

Reply:
> `typename VectorType` : `VectorType` makes confusion reading the code. I was searching `VectorType` at first sight.
> It would be better to use `VECTORTYPE`

Done.